src/unpackertool/plugins/upx/upx_stub_signatures.h

```diff
@@ -12,6 +12,8 @@
 #include "unpackertool/plugins/upx/upx_stub.h"
 #include "retdec/unpacker/signature.h"
 
+using namespace retdec::utils;
```

Comment:
Same as the comment I made earlier. Never put `using namespace` in header files.